packages/rum-core/src/common/utils.js

```diff
@@ -143,8 +143,13 @@ function isPlatformSupported() {
  */
 function setLabel(key, value, obj) {
   if (!obj || !key) return
-  var skey = removeInvalidChars(key)
-  if (value) {
+  const skey = removeInvalidChars(key)
+  let valueType = typeof value
+  if (
```

Comment:
This breaks the previous assumption when the value is `null`, We need to add a test when value is null.